client/blocks/reader-recommended-sites/style.scss

```diff
@@ -44,55 +44,55 @@
 	@include breakpoint( '>960px' ) {
 		align-items: flex-end;
 		flex-direction: column;
-		margin-top: -40px;
 	}
 
 	> :last-child {
 		width: 100%;
+		height: 100%;
 	}
 
 	.reader-recommended-sites__recommended-site-dismiss {
-		width: 30px;
+		justify-content: flex-end;
+		position: absolute;
+		top: 16px;
+		right: 16px;
 
 		@include breakpoint( '>960px' ) {
-			display: flex;
-			justify-content: flex-end;
+    		top: 0;
```

Comment:
Just a minor formatting nit: should these (and a few of the lines below) be indented? :)